src/ModelFactory.h

```diff
@@ -79,6 +80,7 @@ class ModelFactory
 {
 public:
     static std::unique_ptr<ModelConfig> Create(const std::string& name, const SNR&);
```

Comment:
remove *what* API?  I like ModelFactory::Create, it is convenient.  Is there a preferred alternative?